vertica_python/tests/integration_tests/test_cursor.py

```diff
@@ -204,13 +206,28 @@ def test_copy_with_string(self):
             res_from_cur2 = cur2.fetchall()
             self.assertListOfListsEqual(res_from_cur2, [[2, 'bar']])
 
-    def test_copy_with_file(self):
-        with tempfile.TemporaryFile() as tmpfile, self._connect() as conn1, self._connect() as conn2:
-            if _os.name != 'posix' or _os.sys.platform == 'cygwin':
-                f = getattr(tmpfile, 'file')
```

Comment:
Any reason to delete these two lines? Are you sure there will be no regression?

What I see in https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryFile is
> The returned object is a true file object on POSIX platforms. On other platforms, it is a file-like object whose file attribute is the underlying true file object.

Reply:
Oops, I added a blurb about this in the [commit message](https://github.com/vertica/vertica-python/pull/326/commits/60a73f4272ef3ec409d075086434e2c66ddcecb4) but forgot to describe it here as well.

In all Python versions since 2.6 (and possibly before 2.6 but I didn't check), [whenever `os.name != 'posix' or sys.platform == 'cygwin'` is `True`, `TemporaryFile` is just an alias of `NamedTemporaryFile`](https://github.com/python/cpython/blob/v3.8.0/Lib/tempfile.py#L552-L555).  The test I added will fail if for some reason we cannot perform a `COPY` with `NamedTemporaryFile` or `TemporaryFile`, so if there is a regression the newly-added test will catch it.